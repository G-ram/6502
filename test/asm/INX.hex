a2 08 e8 ca 8e 01 01 
