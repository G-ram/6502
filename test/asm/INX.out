A: 00
X: 08
Y: 00
P: 00010000
S: 20
PC: 607
100 : 00
101 : 08
